Weird build failure due to formatting error reported by `flake8` - didn't get such error when running `python3 setup.py test` locally.

```
========================== 296 passed in 1.96 seconds ==========================
flake8 create: /home/travis/build/typesafehub/conductr-cli/.tox/flake8
flake8 installdeps: flake8, pep8-naming
flake8 inst: /home/travis/build/typesafehub/conductr-cli/.tox/dist/conductr-cli-0.46.zip
flake8 installed: argcomplete==1.6.0,arrow==0.8.0,conductr-cli==0.46,flake8==3.2.0,jsonschema==2.4.0,mccabe==0.5.2,pager==3.3,pep8-naming==0.4.1,prettytable==0.7.2,pycodestyle==2.2.0,pyflakes==1.3.0,Pygments==2.1.3,pyhocon==0.2.1,pyparsing==2.0.3,python-dateutil==2.6.0,requests==2.11.1,requests-toolbelt==0.7.0,six==1.10.0,toml==0.9.2
flake8 runtests: PYTHONHASHSEED='2538034910'
flake8 runtests: commands[0] | flake8 --ignore=E501 .
./setup.py:59:1: E305 expected 2 blank lines after class or function definition, found 1
./conductr_cli/conduct.py:12:1: E305 expected 2 blank lines after class or function definition, found 1
./conductr_cli/sandbox.py:12:1: E305 expected 2 blank lines after class or function definition, found 1
./conductr_cli/shazar.py:12:1: E305 expected 2 blank lines after class or function definition, found 1
./dcos/subcommand.py:394:1: E305 expected 2 blank lines after class or function definition, found 1
ERROR: InvocationError: '/home/travis/build/typesafehub/conductr-cli/.tox/flake8/bin/flake8 --ignore=E501 .'
```

The error all occurs in the files not touched by this PR. Not sure why this is happening now.

Will try rebasing with master.

--- a/conductr_cli/conduct.py
+++ b/conductr_cli/conduct.py
@@ -9,5 +9,6 @@ def main_method():
 def run():
     main_handler.run(main_method)
 
+
 if __name__ == '__main__':
     run()

--- a/conductr_cli/sandbox.py
+++ b/conductr_cli/sandbox.py
@@ -9,5 +9,6 @@ def main_method():
 def run():
     main_handler.run(main_method)
 
+
 if __name__ == '__main__':
     run()

--- a/conductr_cli/shazar.py
+++ b/conductr_cli/shazar.py
@@ -9,5 +9,6 @@ def main_method():
 def run():
     main_handler.run(main_method)
 
+
 if __name__ == '__main__':
     run()

--- a/dcos/subcommand.py
+++ b/dcos/subcommand.py
@@ -391,6 +391,7 @@ def uninstall(package_name):
 
     return False
 
+
 BIN_DIRECTORY = 'Scripts' if util.is_windows_platform() else 'bin'
 
 

--- a/setup.py
+++ b/setup.py
@@ -56,6 +56,7 @@ class Tox(test):
         errno = tox.cmdline(args=args)
         sys.exit(errno)
 
+
 setup(
     name='conductr-cli',
     version=__version__,

